client/app/components/dynamic-parameters/DateRangeParameter.jsx

```diff
@@ -38,6 +38,12 @@ const DYNAMIC_DATETIME_OPTIONS = [
   ...DYNAMIC_DATE_OPTIONS,
 ];
 
+const widthByType = {
+  'date-range': 270,
+  'datetime-range': 320,
+  'datetime-range-with-seconds': 370,
+};
```

Comment:
You're right about the clear button position.
Here's a new set of widths with a perfect layout:
```js
const widthByType = {
  'date-range': 294,
  'datetime-range': 352,
  'datetime-range-with-seconds': 382,
};
```
And also:
```css
.redash-datepicker .ant-calendar-picker-clear {
  background: transparent;
}
```